src/components/presentational/Section/Section.tsx

```diff
@@ -7,6 +7,7 @@ export interface SectionProps {
 	children?: React.ReactNode;
 	className?: string;
 	noTopMargin?: boolean;
+	noBottomMargin?: boolean;
```

Comment:
I was just mirroring the existing `noTopMargin` prop and wanted to retain backward compatibility (including a bottom border by default).

Do you have a suggestion for what to name it instead?

Reply:
Yeah this was just because I very commonly wanted to exclude the top margin; I think view builder has noTopMargin set universally since all other cards/sections have a margin by default.

We should probably just follow the pattern from TextBlock though and pass through ```style```:
https://github.com/CareEvolution/MyDataHelpsUI/blob/main/src/components/presentational/TextBlock/TextBlock.tsx#L11

And have this in the view builder like we do for text:
![Screenshot 2024-02-08 at 2 16 33 PM](https://github.com/CareEvolution/MyDataHelpsUI/assets/1643171/f070573e-d875-4397-8a0c-987b1397e453)

I would keep noTopMargin universally on section in the viewbuilder though so by default it's still 0 top margin, but can be overridden with^

Reply:
Done.